docs/root/configuration/other_features/dlb.rst

```diff
@@ -0,0 +1,42 @@
+.. _config_connection_balance_dlb:
+
+DLB Connection Balancer
+=======================
+
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.dlb.v3alpha.Dlb>`
+
+
+This connection balancer extension provides Envoy with low latency networking by integrating with `Intel DLB <https://networkbuilders.intel.com/solutionslibrary/queue-management-and-load-balancing-on-intel-architecture>`_ through the libdlb library.
+
+The DLB connection balancer is only included in :ref:`contrib images <install_contrib>`
+
+Example configuration
+---------------------
+
+.. code-block:: yaml
+
+  static_resources:
+    listeners:
+      connection_balance_config:
+        extend_balance:
+          name: envoy.network.connection_balance.dlb
+          typed_config:
+            "@type": type.googleapis.com/envoy.extensions.dlb.v3alpha.Dlb
+
+How it works
+------------
+
+If enabled, the DLB connection balancer will
+- attache DLB hardware
+- create a queue for balancing
+- create one port to send and one port to receive for each worker thread
+- create one eventfd for each worker thread and attach each eventfd to corresponding customer
+- register each eventfd to corresponding customer and DLB hardware
+
+When new connections come, one worker thread will accept it and send it to DLB hardware. DLB hardware
+does balancing then trigger one worker thread to receive via libevent.
```

Comment:
```suggestion
When a new connection is established, a single worker thread accepts it notifies the DLB hardware. The DLB hardware performs the load balancing and in turn triggers a worker thread via libevent.
```

not sure if my suggestion is accurate - i didnt fully understand the original sentence

Reply:
I mean:
> When a new connection is established, a single worker thread accepts it and send its' pointer to the DLB hardware. The DLB hardware performs the load balancing and in turn triggers a worker thread via libevent.

Is it more clear?